### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by lukesleeman**
This updates our readme file with more information about the app and how to contribute.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,47 @@
-# wh_covid19_app
+# EGBA
 
-A tool to safeguard the welfare of WH anaesthetists during COVID19 and provide instant access to the most up-to-date information for time-critical procedures to ensure optimal patient care. 
+> Everything
```
Comment (Mattro) [inline README.md]: "Everythings"

**Review by Mattro — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lukesleeman) [reply README.md]: Fixed

File: README.md [hunk 2]
```diff
@@ -1,16 +1,47 @@
...
+
+Matt Kelsh - Design
+
+## License
+
+This project is licensed under the Apache 2.0 - see the `LICENSE-2.0.txt` file for details.
+
+## Acknowledgments
+
+🎩 Huge hat tip to everybody providing advice, helping out, etc including: Gregg Miller, Matt Kelsh, Mike Hughes,  Alessandro Favero Alessandro Favero Alessandro Favero, Maksim Lin, Brett Morgan,
```
Comment (alessandrofavero) [inline README.md]: Multiple repetitions on the last line.

**Review by alessandrofavero — CHANGES_REQUESTED** [review]
Multiple repetitions on the last line. To be removed.

File: README.md (hunk 2 above)
Comment (lukesleeman) [reply README.md]: 😂 This is now fixed

**Review by alessandrofavero — APPROVED** [review]

**Pull request merged**